core/lib/spree/testing_support/factories/event.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :event, class: Spree::Webhooks::Event do
+    enabled { true }
+    name { 'resource.create' }
```

Comment:
```suggestion
    name { 'order.create' }
```

let's use always real-world scenarios, btw on the topic of Order we will also need to track order-specific transitions:

`order.cancel`
`order.resume`
`order.complete`

same for shipment and payment